### ISSUE TITLE
Custom Transformers should be loaded independently from --transform option

### ISSUE THREAD

**Issue by jyrlindr** [MarketSquare/robotframework-tidy#476]
It should be possible to take custom transforms in to use without disabling all the default transformers.

Maybe there could be a new option that enables to define custom transformers to be used in addition to default ones.

Better way to do this would be that there's an option to define location(s) where the tool would try to find all the transformers and load them all. f.g.
> laod-transformers = ["myproject.transformers"]

With such approach I could just add new transformers into that folder and take them into use without modifying config files in each and every project I want to use my custom transformers.

**Comment by bhirsz** [comment]
I agree. So we need:

- new option that would load transformers on top of default ones, for now let's call it ``--load-transformers`` (your naming sense is already better than mine :) )
- a way of loading transformers from modules/directories
- and since we want to load multiple transformers, a easy way to define their order. Maybe we can reuse TRANSFORMERS = [name1, name2...] in init file approach that is used in Robotidy? Of course such conventions would need to be well documented.


For example:

```
ImportantCompany.Robotidy.Transformers/
   __init__.py
  TransformerA.py
  TransformerB.py

```

we could follow Robocop approach where all rules had to be imported in init file:

```
__init__.py content:
from ImportantCompany.Robotidy.Transformers.TransformerA import TransformerA
from ImportantCompany.Robotidy.Transformers.TransformerA import TransformerB

TRANSFORMERS = [
    "TransformerB",
   "TransformerA"
]
```

**Comment by bhirsz** [comment]
I split it into the separate features because the first one is fairy easier to be added than the second, so I could do it in separate releases if needed.

**Comment by bhirsz** [comment]
I did MVP of the first feature and it works quite well. Now I'm working on adding tests for it. 

It will work the same as transform and configure do you will be able to pass configuration after :. It will also integrate nicely so you load custom transformers and then use --configure to configure them. By default robotidy will run all default transformers and then custom ones. But it will respect --transform option so will be possible to select default transformers before tuning custom ones.

**Comment by bhirsz** [comment]
Linked the PR with loading the whole modules/directories  - after the next release it will be possible to import multiple transformers:

```
robotidy --load-transformers MyCustomModule.transformers
```

**Comment by bhirsz** [comment]
This change was released together with Robotidy 4.0. However I recommend using Robotidy 4.0.1 when it is (soon) released since I also introduced regression.
